### ISSUE TITLE
Enabling filetype and plugin highlights

### ISSUE THREAD

**Issue by kristijanhusak** [lifepillar/vim-gruvbox8#1]
Hi,

i noticed that highlights for filetype and plugin exists in the templates, but they are not part of the colorscheme.
Is there a way to add it to the colorscheme, except building the colorscheme by myself?

**Comment by lifepillar** [comment]
No, you have to rebuild the colorscheme. But I may add them as an option.

**Comment by kristijanhusak** [comment]
That would be great. I really like gruvbox, but it has many issues and i'ts not maintained.

**Comment by lifepillar** [comment]
>I really like gruvbox, but it has many issues and i'ts not maintained.

Maybe you may chime in at https://github.com/vim/vim/pull/2573.
